src/test/java/unit/GcpNotebookControlled.java

```diff
@@ -204,13 +207,19 @@ void overrideLocationAndInstanceId() throws IOException {
             "--name=" + name,
             "--new-name=" + newName,
             "--new-description=" + newDescription,
-            "--new-metadata=" + newMetadata);
+            "--new-metadata=" + newMetadata1 + "," + newMetadata2);
 
     // check that the properties match
     // the metadata supports multiple entries, we can't assert on the metadata because it's not
     // stored in or accessible via Workspace Manager.
     assertEquals(newName, updatedNotebook.name, "create output matches name");
     assertEquals(newDescription, updatedNotebook.description, "create output matches description");
+    assertTrue(
+        updatedNotebook.metadata.toString().contains(newMetadata1),
```

Comment:
i think i prefer `assertEquals(metadata1, updatedNotebook.metadata.get(NewMetadata1))` than converting map to string.

Reply:
➕